subsystem/cluster_waiter_test.go

```diff
@@ -175,7 +175,7 @@ var _ = Describe("Cluster creation", func() {
 				}
 			`)
 
-		Expect(terraform.Apply()).To(BeZero())
+		Expect(terraform.Apply()).NotTo(BeZero())
 		resource := terraform.Resource("rhcs_cluster_wait", "rosa_cluster")
```

Comment:
Does it possible to add a test that cluster is still installing and the waiter will be failed?

Reply:
This is the test which reproduce the issue
The cluster wait is finished with timeout and state="waiting"
This is why added the change to expect result not to be zero